chainer/link_hook.py

```diff
@@ -62,7 +62,7 @@ class LinkHook(object):
     :meth:`~chainer.LinkHook.forward_preprocess`
     (resp. :meth:`~chainer.LinkHook.forward_postprocess`)
     of all link hooks registered to this link are called before (resp. after)
-    :meth:`Link.forward() <chainer.Link.forward>` method of the link.
+    ``forward()`` method of the link.
```

Comment:
Instead of disabling cross reference, how about implementing `Link.forward` which just raises `NotImplementError`?

Reply:
I think having a link that when clicked provides no information disappoints the user. I think it's better to leave it unlinked, since the user has to provide the forward() method himself.

Reply:
By implementing `Link.forward`, we can write documentation there.

Reply:
I'd like to keep it annotated (``:meth:`~chainer.Link.forward` `` looks better to me).  Being a dead link is informative here.

If `Link.forward` is defined, its docstring could give info to users.

A `Link` subclass often defines `forward`, but not always.  For example, when I define two equally-important calls, neither of them is named `forward`.